app/init/README.md

```diff
@@ -0,0 +1,10 @@
+# Init
```

Comment:
Why create the init container, instead of calling each Cloud Run Job and cache clear sequentially via the HTTP provider directly?